src/managers/input/Touch.cpp

```diff
@@ -24,6 +28,25 @@ void CInputManager::onTouchDown(wlr_touch_down_event* e) {
         return;
     }
 
+    if (*PSWIPETOUCH && (!m_pFoundLSToFocus || m_pFoundLSToFocus->layer == ZWLR_LAYER_SHELL_V1_LAYER_BACKGROUND)) {
+        const auto PWORKSPACE = g_pCompositor->getWorkspaceByID(PMONITOR->activeWorkspace);
+        const bool VERTANIMS  = PWORKSPACE->m_vRenderOffset.getConfig()->pValues->internalStyle == "slidevert" ||
+            PWORKSPACE->m_vRenderOffset.getConfig()->pValues->internalStyle.starts_with("slidefadevert");
+        // TODO: support no_gaps_when_only?
+        const auto target_portion = (*PGAPSOUT + *PBORDERSIZE) / (VERTANIMS ? PMONITOR->vecSize.y : PMONITOR->vecSize.x);
```

Comment:
no snake_case, we use camelCase

additionally, const values we SHOUT

so `TARGETPORTION` and `POSITION`